ROMFS/px4fmu_common/init.d/2107_fw_AERTA

```diff
@@ -0,0 +1,28 @@
+#!nsh
+#
+# @name Standard AERTA Plane
```

Comment:
I didn't think of that. Just too lazy to move wires around to use the AAERT config...
So I'll delete this config and modify the AERT config without renaming it.